python/dolfinx/mesh.py

```diff
@@ -439,7 +440,18 @@ def create_mesh(
     return Mesh(mesh, domain)
 
 
-def create_submesh(msh, dim, entities):
+def create_submesh(msh: Mesh, dim: int, entities: npt.NDArray[np.int32]):
+    """Create a mesh based on a subset of entities from an existing mesh.
+
+    Args:
+        mesh: Input mesh
+        dim: Topological dimension of the entities to extract
+        entities: Indices of entities in ``msh`` to extract
+    Returns:
+        A quadruplet containing the submesh and three maps from the submesh to
```

Comment:
The return type is a `tuple`. Use (0), (1), etc, to be precise on what each element is. 'map' needs to be clear on what it maps from and to.